tools/keycloak/sync/realm.test.json

```diff
@@ -39,15 +39,15 @@
         "ImplicitFlowEnabled": false,
         "DirectAccessGrantsEnabled": true,
         "AuthorizationServicesEnabled": false,
-        "RootUrl": "https://pims-app-test-3cd915-dev.apps.silver.devops.gov.bc.ca",
+        "RootUrl": "https://pims-app-3cd915-dev.apps.silver.devops.gov.bc.ca",
         "RedirectUris": [
-          "https://pims-app-test-3cd915-dev.apps.silver.devops.gov.bc.ca/*",
-          "https://tst-pims.th.gov.bc.ca/*"
+          "https://pims-app-3cd915-dev.apps.silver.devops.gov.bc.ca/*",
+          "https://dev-pims.th.gov.bc.ca/*"
```

Comment:
@devinleighsmith just confirming that we want to point the TEST kc realm to the DEV api instance? It would seem that only the service account needs to be changed...